client/assets/components/navbar.scss

```diff
@@ -0,0 +1,27 @@
+@import url('http://fonts.googleapis.com/css?family=Abel');
```

Comment:
### Nitpick

Since `Abel` is the global font for the project, why not set this in the base stylesheet every other stylesheet inherits from. Also this will help in the event we need to change the font globally, we won't need to start going to different places and changing it.

Also, it's safe to even set the `body` tag to use the `Abel` font since that's what the project design is based on.